spec/features/jobs/user_see_all_tags_for_each_job_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "User visits individual job page" do
+  scenario "and sees all tags for that job" do
+    company = Company.create!(name: "ESPN")
+    job = company.jobs.create!(title: "Developer", level_of_interest: 70, city: "Denver")
+
+    visit company_job_path(company, job)
+
+    fill_in "tag[title]", with: "resume sent"
+    click_on "Add Tag"
+    fill_in "tag[title]", with: "interview scheduled"
+    click_on "Add Tag"
+    fill_in "tag[title]", with: "contacted"
+    click_on "Add Tag"
+
```

Comment:
While filling out this form looks like good setup, be careful of giving yourself more work than need be. The user stories didn't ask for this type of interaction, so nixing it could have saved you a lot of time.